docgen/layouts/common/header.pug

```diff
@@ -1,24 +1,78 @@
-mixin navListItem(link, title, navGroup, currentNavGroup)
-  - var isCurrentNavGroup = currentNavGroup == navGroup;
-  - var itemCssClass = isCurrentNavGroup ? ['ac-nav-menu-list-item', 'active'] : ['ac-nav-menu-list-item'];
-  li(class=itemCssClass)
-    a(data-path=link, href=link, title=title)=title
+.cm-navigation
+  .cm-navigation__brands
+    .cm-navigation__brands--algolia
+      a#algoliaLogo(href=h.publicPath('https://algolia.com'))
+        svg(width='91px', height='29px', viewbox='0 0 91 29', version='1.1', xmlns='http://www.w3.org/2000/svg', xmlns:xlink='http://www.w3.org/1999/xlink')
+          g#Page-1(stroke='none', stroke-width='1', fill='none', fill-rule='evenodd')
+            g#logo-algolia(fill='#000000')
+              path#Shape(d='M54.778,11.404 L53.584,15.472 L57.429,13.446 C56.869,12.461 55.917,11.716 54.777,11.404 L54.778,11.404 Z M47.448,8.07 C46.856,7.524 45.893,7.524 45.298,8.07 L45.031,8.317 C44.438,8.863 44.438,9.747 45.031,10.294 L45.317,10.557 C45.937,9.649 46.709,8.855 47.603,8.211 L47.45,8.071 L47.448,8.07 Z M56.188,6.796 C56.188,6.766 56.198,6.736 56.198,6.703 L56.198,5.966 C56.198,5.153 55.538,4.493 54.726,4.493 L52.15,4.493 C51.34,4.493 50.68,5.153 50.68,5.966 L50.68,6.69 C52.487,6.182 54.403,6.22 56.187,6.797 L56.188,6.796 Z M45.456,15.724 C45.456,20.177 49.096,23.788 53.584,23.788 C58.074,23.788 61.712,20.178 61.712,15.725 C61.712,11.272 58.072,7.66 53.584,7.66 C49.094,7.66 45.456,11.272 45.456,15.724 L45.456,15.724 Z M53.584,9.964 C56.784,9.964 59.389,12.548 59.389,15.724 C59.389,18.9 56.784,21.484 53.583,21.484 C50.383,21.484 47.778,18.9 47.778,15.724 C47.778,12.548 50.381,9.964 53.582,9.964 L53.584,9.964 Z')
```

Comment:
I don't know how to solve it but we will have to do a second pass on this so that the SVG files are not inlined into the pug file